File: Data Structures & Algorithms/concatenation-of-array/submission-2.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int poop [] = new int[2*nums.length];
        for(int i =0; i<nums.length; i++){
            poop[i] = nums[i];
            poop[i+(nums.length)]=nums[i];
        }
        return poop;
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-2.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        res = [1] * (len(nums))
        prefix = 1
        postfix = 1
        for x in range(len(nums)):
            res[x] = prefix
            prefix *= nums[x]
        for j in range(len(nums)-1,-1,-1):
            res[j] *= postfix
            postfix *=nums[j]
        return res
            
        